invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/BookDetails.scss

```diff
@@ -1,15 +1,17 @@
-.list {
-  overflow-wrap: break-word;
-}
+.ui.segment.book-metadata {
+  .list {
+    overflow-wrap: break-word;
+  }
 
-.ui.circular.button {
-  margin: 10px;
-}
+  .ui.circular.button {
+    margin: 10px;
+  }
 
-.row {
-  margin-right: 20px;
-}
+  .row {
+    margin-right: 20px;
+  }
 
-.ui.labeled.button {
-  margin: 10px;
+  .ui.labeled.button {
```

Comment:
this is fixed in the other PR right? Because it will affect all the buttons with class labeled

Reply:
It's fixed in this PR, I nested them inside .ui.segment.book-metadata